### DISCUSSION TITLE
Settings Organization

### DISCUSSION THREAD

**Discussion by karpalakllc** [Laravel-Backpack/community-forum#835] [General]
Hello 👋
I would like to ask about your opinion on what's the best way to organize all of my settings.
They are all used for ecommerce websites. Below you can see all of my setting tabs that I currently have.
<img width="236" alt="image" src="https://github.com/Laravel-Backpack/community-forum/assets/150701400/06c68d55-570d-457c-a9c8-e43f3c0c8db1">

So let's say when I go to website settings, inside you can find all kind of configuration related to the website header, footer, SEO, loyalty program, mobile settings etc.
And as you can see on the screenshot, there are 8 pages. 
<img width="1450" alt="image" src="https://github.com/Laravel-Backpack/community-forum/assets/150701400/b9a42616-22cd-4644-bd99-3d8e3f9cc1cc">

The same situation is in Shop Settings, Food Settings, Marketing Settings...What would be the most convenient and practical way to group and organize all of these settings? I am thinking something like, for example when I click on Website Settings, I should have multiple tabs like in the screenshot below, or maybe use the backpack filters but place them vertically if its possible.
![image](https://github.com/Laravel-Backpack/community-forum/assets/150701400/d61ef182-490f-444b-87a9-30998083d2f8)

My current developers are new to backpack and they are not very familiar with all of the backpack features so they can't tell me what would be the best way to achieve this.

Thank you.

**Comment by jcastroa87** [comment]
Hello @karpalakllc the most appropriate option will be the one that takes the user the least time to find, for this, you can use nav-title to divide the configurations section and then leave them grouped internally, the other way is the one you propose through the use of tabs, but you must take into account if it is appropriate for you to use tabs to update all the configurations of the groups. I recommend talking to the platform users to choose the most appropriate one since you have several alternatives to achieve this.

Cheers.
